code/modules/mob/living/carbon/life.dm

```diff
@@ -90,13 +88,17 @@
 //Third link in a breath chain, calls handle_breath_temperature()
 /mob/living/carbon/proc/check_breath(datum/gas_mixture/breath)
```

Comment:
I could probably just `return` on this proc; we don't have any carbons that would utilize this, as far as I know; brains don't breathe, and aliens have their own `check_breath`.